indigo_app/static/javascript/indigo/models.js

```diff
@@ -110,27 +110,10 @@
         }
       }
 
-      // ensure attachment ids are correct, we could have deleted an attachment
-      if (oldNode.tagName === 'attachment') {
```

Comment:
I don't think we can remove this. Slaw doesn't know if an attachment has been deleted during a quick edit, which requires eIds of subsequent attachments to be re-numbered. The only place that has sufficient context to do that is here.